task_8_9_10.py

```diff
@@ -0,0 +1,24 @@
+from utils import print_intro
+
+print_intro()
+
+
+def get_position():
+    while True:
+        position = input('Enter the position of the chess piece (e.g. a1): ')
+        if len(position) != 2 or position[0] not in 'abcdefgh' or position[1] not in '12345678':
```

Comment:
'abcdefgh'
'12345678'
https://code-basics.com/languages/python/lessons/magic-numbers

Reply:
if position in (''.join(i) for i in product('abc', '123')):